index.html

```diff
@@ -0,0 +1,285 @@
+<!DOCTYPE html>
+<div lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Maher</title>
+    <link rel="stylesheet" href="css/style.css">
+    <!-- link swiper css -->
+    <link rel="stylesheet" href="CSS/swiper-bundle.min.css">
+    <!-- BOX ICONS -->
+</head>
+
+    <!-- Header -->
+        <!-- nav -->
+        <header>
+        <div class="nav container">
+            <!-- Logo -->
+          <nav class="navbar">
+            <a href="#">Maher</a>
+          </nav>
+          <div class="end">
+            <img class="endnavbar" src="./img/Union.png" alt="Menu">
```

Comment:
As the previous reviewer stated, please wrap the hamburger menu in a nav tag instead of the div tag since it is a navigation menu and it is a good convention to do that.